### DISCUSSION TITLE
many-to-many relationship with addional pivot columns

### DISCUSSION THREAD

**Discussion by ndum** [statamic-rad-pack/runway#288] [Help]
Hello

I'm working with a many-to-many relationship in Laravel/Statamic and have added an additional column to the pivot table. Here is my User model:

```
namespace App\Models;

use Illuminate\Database\Eloquent\Model;

class User extends Model
{
    public function roles()
    {
        return $this->belongsToMany(Role::class, 'role_user')
                    ->withPivot('assigned_at')
                    ->withTimestamps();
    }
}
```

I would like to know how to display the additional pivot column "assigned_at" with runway. Is this possible? Unfortunately I did not find anything in the documentation.

Thank you for your assistance!

**Comment by duncanmcclean** [comment]
Currently, there's no way to show pivot data columns in Runway. I can't think of an easy way to do it or even where to display it.

If you find a solution, I'd welcome a pull request.
